packages/connection/src/class-utils.php

```diff
@@ -18,24 +18,16 @@ class Utils {
 	const DEFAULT_JETPACK__API_BASE    = 'https://jetpack.wordpress.com/jetpack.';
 
 	/**
-	 * Some hosts disable the OpenSSL extension and so cannot make outgoing HTTPS requests.
-	 * This method sets the URL scheme to HTTP when HTTPS requests can't be made.
+	 * This method used to set the URL scheme to HTTP when HTTPS requests can't be made.
+	 * Now it returns the exact same URL you pass as an argument.
 	 *
 	 * @param string $url The url.
-	 * @return string The url with the required URL scheme.
+	 * @return string The exact same url.
+	 *
+	 * @deprecated 9.1.0 Jetpack can't function properly on servers that don't support outbound HTTPS requests.
 	 */
 	public static function fix_url_for_bad_hosts( $url ) {
```

Comment:
Minor but let's also remove the reference to this file `* @see Utils::fix_url_for_bad_hosts()` in the compat package.

Reply:
Cleaned up 👍